pyccel/ast/builtin_methods/list_methods.py

```diff
@@ -48,7 +49,7 @@ class ListAppend(PyccelInternalFunction):
     _shape = None
     _order = None
     _rank = 0
-    _precision = -1
+    _precision = None
```

Comment:
Why did you make this change?

Reply:
The call to append does not result in anything, so I think it's not going to require space in memory and hence has no precision